Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -134,7 +134,7 @@ PlayerSettings:
     16:10: 1
     16:9: 1
     Others: 1
-  bundleVersion: 0.1
+  bundleVersion: 97
```

Comment:
this isn't needed, but doesn't hurt. It's just the current latest build in live app testing

Reply:
For Amazon only right? I think this would likely get outdated soon, considering we reuse the same app in multiple sdks for amazon... NABD though.

Reply:
yeah, this was just me being lazy to roll this change back, and since it doesn't make a difference either way